civis/ml/tests/test_model.py

```diff
@@ -123,10 +127,11 @@ def foo(arg):
 
 @mock.patch.object(_model.cio, 'file_to_civis', return_value=-11)
 def test_stash_local_data_from_file(mock_file):
-    with tempfile.TemporaryDirectory() as tempdir:
-        fname = os.path.join(tempdir, 'filename')
+    with tempfile.NamedTemporaryFile() as tempfname:
```

Comment:
Ah, right. Yeah, I'm going to punt on this. I'll see about adding appveyor support to make sure we're covered for CI on windows soon-ish.